doc/source/cheatsheet/cheatsheet.qmd

```diff
@@ -132,12 +127,12 @@ wb.run_script_file("project_workflow.wbjn", log_level="info")
 
 # Upload and download files
 
-Using **upload_file()** and **download_file()** methods, you have the ability to transfer data files to the server(upload) and retrieve them from the server (download).
+Employ **upload_file()** and **download_file()** methods to transfer data files to and from the server.
```

Comment:
```suggestion
Use the ``upload_file()`` and ``download_file()`` methods to transfer data files to and from the server.
```
Keep the verbs simple and use code entity tags.

Reply:
Done. Thanks!